staking_deposit/exit_transaction.py

```diff
@@ -0,0 +1,66 @@
+
+import json
+import os
+import time
+from typing import Any, Dict, List
+from py_ecc.bls import G2ProofOfPossession as bls
+
+from staking_deposit.settings import BaseChainSetting
+from staking_deposit.utils.constants import DEFAULT_EXIT_TRANSACTION_FOLDER_NAME
+from staking_deposit.utils.ssz import (
+    SignedVoluntaryExit,
+    VoluntaryExit,
+    compute_signing_root,
+    compute_voluntary_exit_domain,
+)
+
+
+def exit_transaction_generation(
+        chain_settings: BaseChainSetting,
+        signing_key: int,
+        validator_index: int,
+        epoch: int) -> SignedVoluntaryExit:
+    message = VoluntaryExit(
+        epoch=epoch,
+        validator_index=validator_index
+    )
+
+    domain = compute_voluntary_exit_domain(
+        fork_version=chain_settings.CURRENT_FORK_VERSION,
```

Comment:
Hardcode to CAPELLA_FORK_VERSION following https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/beacon-chain.md?plain=1#L411

Reply:
ack